.circleci/checkout_merge_commit.sh

```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
 
 
-# Add `master` branch to the update list.
-# Otherwise CircleCI will give us a cached one.
```

Comment:
Please don't make changes outside of the recipes dir in a recipe PR. Please put this in as a separate PR. Thanks!

Reply:
I'll make a new PR.